ArtOfIllusion/src/artofillusion/LayoutWindow.java

```diff
@@ -45,30 +56,31 @@ Changes copyright (C) 2017 by Petri Ihalainen
   SceneViewer theView[];
   BorderContainer viewPanel[];
   FormContainer viewsContainer;
-  FormContainer centerContainer;
+  
```

Comment:
Removing this breaks the build - it is used in multiple places!

```suggestion
  FormContainer centerContainer;
```